bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleRegistryImpl.java

```diff
@@ -15,6 +15,7 @@
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.Set;
+import java.util.function.Predicate;
```

Comment:
Please don't use 1.8 features, because this is a breakable change and the rule engine can't run on jdk  1.7 platforms.